src/blobs.js

```diff
@@ -2084,7 +2085,7 @@ div#${prntid}.ulabel-night *::-webkit-scrollbar-thumb {
    background-color: rgb(51, 51, 51);
 }
 div#${prntid}.ulabel-night *::-webkit-scrollbar-thumb:hover {
-   background-color: rgb(90, 90, 90);
+   background-color: rgb(90, 90, 90);0.739
```

Comment:
?
Supposed to be `rgba` with `0.739` for alpha?

Reply:
i was probably trying to type elsewhere lol, ill delete that